content/en/cloudprem/processing.md

```diff
@@ -0,0 +1,133 @@
+**Helm limitation**: Be aware that Helm imposes a 1 MB size limit on the configuration file due to its underlying etcd storage.  
+
+## Configuration file format
+
+The configuration is a JSON array, where each element represents a processor or a nested pipeline.  
+The order of elements in the array defines the sequential execution order of the processors.  The structure mirrors the output of the Datadog API endpoint `api/v1/logs/config/pipelines`.
+
+
+```json
+[
+ {...processor1},
+ {...processor2}
+]
+```
+
+```json
+[
+    {
+            "type": "pipeline",
+            "id": "U73LOMZ9QG2iM-04OcXypA",
+            "name": "cluster agent logs parsing",
+            "enabled": true,
+            "filter": {
+                "query": "service:cluster-agent"
+            },
+            "processors": [
+                {
+                    "type": "grok-parser",
+                    "id": "xn2WAzysQ1asaasdfakjf",
+                    "enabled": true,
+                    "grok": {
+                        "supportRules": "",
+                        "matchRules": "agent_rule %{date(\"yyyy-MM-dd HH:mm:ss z\"):timestamp} \\| %{notSpace:agent} \\| %{word:level} \\| \\(%{notSpace:filename}:%{number:lineno} in %{word:process}\\) \\|( %{data::keyvalue(\":\")} \\|)?( - \\|)?( \\(%{notSpace:pyFilename}:%{number:pyLineno}\\) \\|)?%{data}\nagent_rule_pre_611 %{date(\"yyyy-MM-dd HH:mm:ss z\"):timestamp} \\| %{word:level} \\| \\(%{notSpace:filename}:%{number:lineno} in %{word:process}\\)%{data}\njmxfetch_rule     %{date(\"yyyy-MM-dd HH:mm:ss z\"):timestamp} \\| %{notSpace:agent} \\| %{word:level}\\s+\\| %{word:class} \\| %{data}"
+                    }
+                },
+                {
+                    "id": "xnsd5oanXq2893utcsQ",
+                    "type": "status-remapper",
+                    "enabled": true,
+                    "sources": ["level"]
+                },
+                {
+                    "type": "date-remapper",
+                    "id": "xn2WAzysQ1asdjJsb9dfb",
+                    "enabled": true,
+                    "sources": ["timestamp"]
+                }
+            ]
+    }
+]
+```
+
+## Compatibility with Datadog SaaS Pipelines
+
+CloudPrem processing aims for parity with Datadog SaaS, allowing direct use of existing SaaS pipeline configurations by ignoring unknown or faulty processors. However, some differences currently exist:
+Some filter queries can’t be parsed: e.g. combining wildcards `@data.message:+*`
+Filter on `message` has a different matching behaviour (also affects e.g. category processor)
+The current implementation is using a regex to grep the word, but it should tokenize the text and try to match the tokens. Phrases are also ignored currently.
+Some grok syntax differs in behaviour: Groks are using regular expressions internally. The regex engines may have slightly different matching behavior.
+Some grok patterns can’t be parsed (e.g. `%{?>notSpace:db.severity}`)
```

Comment:
```suggestion
- Some filter queries can’t be parsed, such as combining wildcards `@data.message:+*`.
- Filter on `message` has a different matching behavior (also affects e.g. category processor).
- CloudPrem uses a regex to grep the word, but it should tokenize the text and try to match the tokens. Phrases are also ignored.
- Groks are using regular expressions internally. The regex engines may have slightly different matching behavior.
- Some grok patterns can’t be parsed (for example, `%{?>notSpace:db.severity}`)
```

Reply:
Are these supposed to be bullet lists? 
Can you clarify this: "Filter on `message` has a different matching behaviour (also affects e.g. category processor)"

Reply:
@PSeitz-dd can you check that?